index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&family=Poppins:ital,wght@0,200;0,500;0,600;0,800;1,100&display=swap" rel="stylesheet">
+    <title>runner world cup</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+</head>
+<body>
+    <span class="upperhead">
+        <div>
+            <img src="./images/Rectangle.png" alt="">
+            <img src="./images/twitter.png" alt="">
```

Comment:
Well done on the top navbar 💯 looks clean, but for better semantics, the `img` tags should have `alt` attributes and also consider improving the Desktop version design, the social icons are a bit too big for now 👇🏽 

## Behance
![Screenshot 2023-04-18 at 22 12 20](https://user-images.githubusercontent.com/45739963/232948636-87576b98-4b4a-4485-84f2-e5ed6a0a9d93.png)

## Your design

![Screenshot 2023-04-18 at 22 12 52](https://user-images.githubusercontent.com/45739963/232948697-0b07df8f-ab8e-4108-8abe-3cab48da2bd9.png)

Just like in the design, consider adding some margins on the top and bottom and reducing the size of the social icons. 👍🏽